### ISSUE TITLE
Smooth camera drag

### ISSUE THREAD

**Issue by bibbybabby** [studio-minus/ppg-bugs-and-requests#2243]
**Feature description**
Smooth dragging that is a toggle

**Feature purpose**
It's nice

**Additional context**
something like minecraft optifine camera

**Comment by DindinYT37** [comment]
I don't think that's really possible on a 2D side-camera, though a small movement interpolation could make it look nicer.

**Comment by bibbybabby** [comment]
> I don't think that's really possible on a 2D side-camera, though a small movement interpolation would probably make it look nicer.

That's what I meant, smooth camera slow down after releasing the drag button, like a car stopping

**Comment by DindinYT37** [comment]
> > I don't think that's really possible on a 2D side-camera, though a small movement interpolation would probably make it look nicer.
> 
> That's what I meant, smooth camera slow down after releasing the drag button, like a car stopping

Yeah, but that wouldn't be like the OptiFine zoom-camera movement from Minecraft. Good that this was cleared up, though.

**Comment by royalealbert27** [comment]
use the cloddymod

**Comment by bibbybabby** [comment]
> use the cloddymod

what if i only want a smooth regular view camera and not the other things

**Comment by royalealbert27** [comment]
> > use the cloddymod
> 
> what if i only want a smooth regular view camera and not the other things

OK then nothing
